java/openjdk11/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+
+name             openjdk11
+version          11
+
+categories       java devel
+maintainers      {breun.nl:nils @breun} openmaintainer
+platforms        darwin
+license          GPL-2
+supported_archs  x86_64
+
+description      Open Java Development Kit 11
+
+long_description Production-ready, free and open-source build of the Java \
+                 Development Kit, an implementation of the Java Standard \
+                 Edition (SE) 11 Platform. OpenJDK is the official reference \
+                 implementation of Java SE. Included components are the \
+                 HotSpot virtual machine, the Java class library and the Java \
+                 compiler. 
+
+homepage         https://jdk.java.net/11/
+master_sites     https://download.java.net/java/GA/jdk11/28/GPL/
+distname         openjdk-${version}+28_osx-x64_bin
+
+checksums        rmd160  c914d38c78b2943c7b47ff2da4e8da89438f96cd \
+                 sha256  6b969d2df6a9758d9458f5ba47939250e848dfba8b49e41c935cf210606b6d38 \
+                 size    182717049
+
+worksrcdir       jdk-${version}.jdk
+
+use_configure    no
+
+build {}
+
+# macOS Java tools expect to find Java virtual machines under /Library/Java/JavaVirtualMachines, which is not under ${prefix}.
+destroot.violate_mtree yes
+
+destroot {
+    set target ${destroot}/Library/Java/JavaVirtualMachines
+    xinstall -m 755 -d ${target}
+    copy ${worksrcpath} ${target}
+}
+
+livecheck.type      regex
+livecheck.url       ${homepage}
+livecheck.regex     "Java Development Kit, version (\\d+(?:\\.\\d+)*(?:-\\d+)?)"
```

Comment:
This livecheck regex doesn't work because "Java Development" is on a separate line from the rest (it doesn't work on the openjdk10 port either).

I would suggest using the distname: `openjdk-(\\d+\\+\\d)_osx-x64_bin`

Reply:
That regex won't match 11.0.1, which is likely to be the next version. I've adjusted the check.